termui/show_bug.go

```diff
@@ -24,6 +24,7 @@ type showBug struct {
 	bug                *cache.BugCache
 	childViews         []string
 	mainSelectableView []string
+	mainTimelineIndex  []int
```

Comment:
I don't think that would be enough because I also need to get the message text from the timeline item. One thing that might work is using the index of the timeline as the `selected` variable.

Is there ever a time when the will be something in `mainSelectableView` that isn't in `Timeline` or vice versa?

Reply:
You can use `Snapshot.SearchTimelineItem` to get the target operation and then get the message.